### PR TITLE
Add addImage test

### PR DESCRIPTION
**Pull request by bsudekum**
Works along side with https://github.com/mapbox/mapbox-gl-native/pull/7610

/cc @jfirebaugh

### CODE REVIEW

File: render-tests/add-image/default/style.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
+{
+  "version": 8,
+  "metadata": {
+    "test": {
+      "width": 64,
+      "height": 64,
+      "center": 0,
```
Comment (jfirebaugh) [inline render-tests/add-image/default/style.json]: Remove the extraneous `center` and `zoom` properties.

File: render-tests/add-image/default/style.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      "center": 0,
+      "zoom": 0,
+      "operations": [
+        [
+          "wait"
```
Comment (jfirebaugh) [inline render-tests/add-image/default/style.json]: This initial `wait` operation isn't necessary.

File: render-tests/add-image/default/style.json [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        ]
+      ]
+    }
+  },
+  "sources": {
+    "geometry": {
+      "type": "geojson",
+      "data": {
```
Comment (jfirebaugh) [inline render-tests/add-image/default/style.json]: Simplify:

```
"data": {
  "type": "Point",
  "coordinates": [0, 0]
}
```

File: render-tests/add-image/default/style.json [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+            "type": "Point",
+            "coordinates": [0, 0]
+          }
+        }]
+      }
+    }
+  },
+  "layers": [
+    {
+      "id": "background",
```
Comment (jfirebaugh) [inline render-tests/add-image/default/style.json]: This background layer isn't necessary. It's fine to have a partially-transparent result.

**Review by jfirebaugh — CHANGES_REQUESTED** [review]

**Comment by jfirebaugh** [comment]
Move the test to `render-tests/runtime-styling/image-add/style.json`.

**Comment by bsudekum** [comment]
@jfirebaugh fixed

**Review by jfirebaugh — APPROVED** [review]

**Pull request merged**